src/Routes.js

```diff
@@ -18,12 +22,15 @@ function Routes() {
       <Switch>
         <Route exact path="/" component={Articles} />
         <Route path="/signup" component={SignUpForm} />
+        <PrivateRoute exact path="/editProfile" component={UpdateProfileForm} isAuthenticated={sessionStorage.getItem('isLoggedIn')} />
+        <PrivateRoute exact path="/viewProfile" component={ViewProfileForm} isAuthenticated={sessionStorage.getItem('isLoggedIn')} />
         <Route path="/login" component={Login} />
-        <Route path="/update-article" component={EditArticle} />
-        <Route path="/create-article" component={CreateArticle} />
-        <Route path="/articles" component={Articles} />
-        <Route path="/:slug" component={Article} />
+        <PrivateRoute path="/update-article" component={EditArticle} isAuthenticated={sessionStorage.getItem('isLoggedIn')} />
+        <PrivateRoute path="/create-article" component={CreateArticle} isAuthenticated={sessionStorage.getItem('isLoggedIn')} />
+        <PrivateRoute path="/articles" component={Articles} isAuthenticated={sessionStorage.getItem('isLoggedIn')} />
+        <PrivateRoute path="/:slug" component={Article} isAuthenticated={sessionStorage.getItem('isLoggedIn')} />
       </Switch>
```

Comment:
@Emmanuel-Dominic this should not be a protected route since users can view a single article even when not logged in.